### PR TITLE
run rustfmt on typecheck/coherence folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by srinivasreddy** [comment]
@nikomatsakis  rollup pls

**Comment by nikomatsakis** [comment]
@srinivasreddy 

> rollup pls

Not sure what you meant by this?

**Comment by srinivasreddy** [comment]
This https://github.com/rust-lang/rust/pull/35892#issuecomment-242837907.

**Comment by nikomatsakis** [comment]
@srinivasreddy I'm still not sure what you meant by that :)

**Comment by srinivasreddy** [comment]
@nikomatsakis  ping.

File: src/librustc_typeck/coherence/orphan.rs [hunk 1]
```diff
@@ -68,7 +77,7 @@ impl<'cx, 'tcx> OrphanChecker<'cx, 'tcx> {
     fn check_item(&self, item: &hir::Item) {
         let def_id = self.tcx.map.local_def_id(item.id);
         match item.node {
-            hir::ItemImpl(.., None, ref ty, _) => {
```
Comment (srinivasreddy) [inline src/librustc_typeck/coherence/orphan.rs]: Isn't  this a bug in rustfmt? cc @nrc

Comment (nrc) [inline src/librustc_typeck/coherence/orphan.rs]: yikes, looks like it

Comment (nrc) [inline src/librustc_typeck/coherence/orphan.rs]: https://github.com/rust-lang-nursery/rustfmt/issues/1150

**Comment by nikomatsakis** [comment]
@srinivasreddy sorry, been AFK due to RustConf prep etc. Seems like we have merge conflicts now =(

**Comment by nikomatsakis** [comment]
Still merge conflicts =(

**Comment by srinivasreddy** [comment]
@nikomatsakis  Done.  :smile:

**Pull request merged**